ui/src/utils/db.js

```diff
@@ -8,6 +8,13 @@ db.version(1).stores({
   userPreferences: '',
 });
 
+db.version(2).stores({
```

Comment:
No need to do this. Feel free to put it directly on the first version.

Reply:
Firstly I am also doing the same things but dexie gives me an error.

Reply:
If you want to remove the error, you'll need to clear the `taskDefinitions` table from IndexDB.

On Chrome:

![screen shot 2019-02-15 at 3 08 20 pm](https://user-images.githubusercontent.com/3766511/52881805-aaca7980-3133-11e9-97ce-bf8134e6d70c.png)


On Firefox:

![screen shot 2019-02-15 at 3 08 41 pm](https://user-images.githubusercontent.com/3766511/52881812-b027c400-3133-11e9-9a48-6e5743bdeabb.png)

Reply:
ok